From 7e2f100f848ccd572536ce4677adb0344683885f Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Sun, 23 May 2021 14:32:46 +0200
Subject: Add missing limits include to fix build with GCC 11

This is not required for Qt 6, since it is indirectly pulled via
qanystrigview.h, but it is for Qt 5 (where qanystrigview does
not exist) and, in any case, it is good to declare all used headers
and not rely on them being implicitly pulled.

Pick-to: 6.1 5.15
Change-Id: I97606ea493e723006759608b7d4c4f00632f340c
Reviewed-by: Albert Astals Cid <albert.astals.cid@kdab.com>
(cherry picked from commit 367293b18ab0d0a0432c1c8ce445fee052e5eee5)
---
 src/qmldebug/qqmlprofilerevent_p.h | 1 +
 1 file changed, 1 insertion(+)

diff --git x/qtdeclarative/src/qmldebug/qqmlprofilerevent_p.h y/qtdeclarative/src/qmldebug/qqmlprofilerevent_p.h
index a7e37d1964..01b2f58f16 100644
--- x/qtdeclarative/src/qmldebug/qqmlprofilerevent_p.h
+++ y/qtdeclarative/src/qmldebug/qqmlprofilerevent_p.h
@@ -48,6 +48,7 @@
 #include <QtCore/qmetatype.h>
 
 #include <initializer_list>
+#include <limits>
 #include <type_traits>
 
 //
-- 
2.33.0

